README.rst

```diff
@@ -67,22 +67,19 @@ PyMedPhys can be installed with:
 
     pip install pymedphys[user]
 
-If you'd like more details around installing PyMedPhys go to the
-`getting started`_ page within the docs.
+Further user installation instructions can be found in the `Quick Start Guide`_.
 
-..
-    TODO: reword:
+The PyMedPhys documentation contains two overarching guides:
 
-The PyMedPhys documentation is split into four categories:
+1. **The Users Guide**: where you can find instructions to `get started`_ with
+   the library and the CLI, in-depth `how-to guides`_ on PyMedPhys' various
+   tools, some `background information`_ on individual PyMedPhys projects as
+   well as the `Technical Reference`_.
 
-1. `Library Users Guide`_: for those building their own Python apps, scripts
-   and other tools who wish to incorporate elements of the PyMedPhys library.
-2. `CLI Users Guide`_: for those who wish to use PyMedPhys' ready-made command
-   line interface (e.g. to help automate existing workflows with minimal
-   programming).
-3. `Contributors Guide`_: for those who wish to make new contributions to
-   either the PyMedPhys library or the PyMedPhys app.
-4. `General`_: Material that may apply to any visitor to PyMedPhys.
+2. **The Contributors Guide**: for those who wish to make new contributions
```

Comment:
This might be confusing, because there aren't any links or descriptive text that says Contributors Guide, instead there is a link entitled "workstation setup guides".

Reply:
Yeah. I guess this is a result of the transition to the new sturcture still being a work-in-progress (see the bottom of the opening post of #1665 and the pages that are listed in the proposed new structure that do not yet exist). 

For now, the side bar (see below) at least matches the categorisation implied by my new wording. Perhaps the bolded text could be converted into a link to the [Overview ](https://docs.pymedphys.com/en/latest/contrib/index.html) page under the Contributors Guide sidebar heading. Unfortunately, this page isn't yet the "intro to contributing" that it ought to be. [I think this was something that @nlambriICH was interested in working on](https://github.com/pymedphys/pymedphys/issues/1665#issuecomment-1153734521), but that may no longer be the case.

![image](https://user-images.githubusercontent.com/19767103/180607754-08f87c7f-8622-40a0-ae21-30641dda958b.png)

Reply:
FWIW, the `docs` folder structure also follows this overarching categorisation. 
![image](https://user-images.githubusercontent.com/19767103/180608128-a8d355e0-cc9f-43ab-abdc-ab76df44847e.png)

Reply:
I'm still interested to rework that part of the documentation, but I didn't find time to work on it... @ddcc3388 has offered to help out (#1588)

Reply:
Great!